README.md

```diff
@@ -131,9 +131,8 @@ the aforementioned validator's node address.
 
 Once you submit your application, the Mezo team will verify your node status and approve 
 your application if everything is in order. Please provide your public IP, your node address, 
-and any custom port settings. If you wish to close the CometBFT RPC port (note that the 
-CometBFT P2P port must remain open), please whitelist the following IP address: `34.57.120.151` 
-so that we can verify your status.
+and any custom port settings. Moreover please ahere to the [central monitoring](#central-monitoring) 
```

Comment:
ahere -> adhere

Reply:
Fixed in 835469f245a9dbfa5650e72b39f0d3895c6336cd